projects/playground/src/app/app.module.ts

```diff
@@ -1,20 +1,34 @@
 import { BrowserModule } from '@angular/platform-browser';
-import { NgModule } from '@angular/core';
+import { NgModule, APP_INITIALIZER } from '@angular/core';
 import { AuthModule } from 'projects/auth0-angular/src/lib/auth.module';
 import { ReactiveFormsModule } from '@angular/forms';
-
 import { AppRoutingModule } from './app-routing.module';
 import { AppComponent } from './app.component';
 import { ProtectedComponent } from './components/protected.component';
 import { UnprotectedComponent } from './components/unprotected.component';
 import { ChildRouteComponent } from './components/child-route.component';
 import { NestedChildRouteComponent } from './components/nested-child-route.component';
+import {
```

Comment:
Possibly intentional, but these newlines are inconsistent with other imports.

Reply:
This has been formatted by Prettier based on line length.